content/en/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md

```diff
@@ -8,24 +8,33 @@ weight: 70
 
 <!-- overview -->
 
-Kubernetes' support for direct integration with Docker Engine is deprecated, and will be removed. Most apps do not have a direct dependency on runtime hosting containers. However, there are still a lot of telemetry and monitoring agents that has a dependency on docker to collect containers metadata, logs and metrics. This document aggregates information on how to detect these dependencies and links on how to migrate these agents to use generic tools or alternative runtimes.
+{{% thirdparty-content %}}
+
+Kubernetes' support for direct integration with Docker Engine is deprecated, and
+will be removed. Most apps do not have a direct dependency on runtime hosting
```

Comment:
This PR has been open long enough for [present tense](https://kubernetes.io/docs/contribute/style/style-guide/#use-present-tense) to be accurate. 😅 
```suggestion
Kubernetes' support for direct integration with Docker Engine is deprecated and
has been removed. Most apps do not have a direct dependency on runtime hosting
```